resources/views/tasks.blade.php

```diff
@@ -38,6 +38,28 @@
                     @endif
                 </div>
             </div>
+<!-----------------------------Date------------------------>
+            <div class="form-group date" data-provide="datepicker">
+                <label for="task" class="col-sm-3 control-label ">Date</label>
+                <div class="col-sm-6">
+                    <input type="date" name="date" class="form-control" data-date-format="mm/dd/yyyy">
+                <div class="input-group-addon">
+                    <span class="glyphicon glyphicon-th"></span>
+                </div>
+                </div> 
+            </div>
+
+            <!--------------------Time---------------------------->
+
+            <div class="form-group date" time-provide="timepicker">
+                <label for="task" class="col-sm-3 control-label ">Time</label>
+                    <div class="col-sm-6">
+                        <input type = 'time' name="time" class="form-control" />  
```

Comment:
why is 'time' in single quotes and also we only need AM, PM, please read the task description